api/libproject/devstart.rb

```diff
@@ -25,38 +25,51 @@
               clouderrorreporting.googleapis.com bigquery-json.googleapis.com}
 DRY_RUN_CMD = %W{echo [DRY_RUN]}
 
+def make_gae_vars(min, max)
+  {
+    "GAE_MIN_INSTANCES" => min.to_s,
+    "GAE_MAX_INSTANCES" => max.to_s
+  }
+end
+TEST_GAE_VARS = make_gae_vars(0, 10)
+
 # TODO: Make environment/project flags consistent across commands, consider
 # using a environment keywords as dict keys here, e.g. :test, :staging, etc.
 ENVIRONMENTS = {
   "local" => {
     :api_endpoint_host => "localhost:8081",
     :cdr_sql_instance => "workbench",
     :config_json => "config_local.json",
-    :cdr_versions_json => "cdr_versions_local.json"
+    :cdr_versions_json => "cdr_versions_local.json",
+    :gae_vars => TEST_GAE_VARS
   },
   "all-of-us-workbench-test" => {
     :api_endpoint_host => "api-dot-#{TEST_PROJECT}.appspot.com",
     :cdr_sql_instance => "#{TEST_PROJECT}:us-central1:workbenchmaindb",
     :config_json => "config_test.json",
-    :cdr_versions_json => "cdr_versions_test.json"
+    :cdr_versions_json => "cdr_versions_test.json",
+    :gae_vars => make_gae_vars(10, 64)
```

Comment:
Again - I would suggest setting min === max to constrain variables for initial profiling. I expect your load test to trigger more task startups which may timeout your requests. Eventually we would want to include the autoscaling in the scope of the load testing but I'd guess it will be too noisy to start there.

I'm guessing you'll be tuning these numbers several times as you go anyways, so I leave it to you how you want to proceed.